examples/controls/src/app.js

```diff
@@ -33,8 +33,8 @@ export default class App extends Component {
         zoom: 3.5,
         bearing: 0,
         pitch: 0,
-        width: window.innerWidth,
-        height: window.innerHeight,
+        width: 500,
```

Comment:
Let's stick to `window.innerWidth` `window.innerHeight` where we can then (or at least whatever the `resize` handler does), since in most cases they seem to get reseted to whatever the `resize` controller does anyway